objcts.js

```diff
@@ -0,0 +1,10 @@
+let numbers = {}
+
+for (var number = 1; number <= 15; number++) {
+  var myKey = number
+  var myValue = number ** 2
```

Comment:
Lets change the name of this from `myValue` to something descriptive e.g `square`